bugs/gh-529-539.txt

```diff
@@ -0,0 +1,2734 @@
+
+        az network lb rule create \
+            --lb-name dsLB_$UBUNTU_RELEASE  \
+            --name dsLBrule_v6_$UBUNTU_RELEASE  \
+            --resource-group $AZURE_RESOURCE_GROUP  \
+            --frontend-ip-name dsLbFrontEnd_v6_$UBUNTU_RELEASE  \
+            --protocol Tcp  \
+            --frontend-port 22  \
+            --backend-port 22  \
+            --probe-name dsProbe_$UBUNTU_RELEASE \
+            --backend-pool-name dsLbBackEndPool_v6_$UBUNTU_RELEASE
+
+        az vm availability-set create \
+            --name dsAVset_$UBUNTU_RELEASE  \
+            --resource-group $AZURE_RESOURCE_GROUP \
+            --location $AZURE_REGION \
+            --platform-fault-domain-count 2  \
+            --platform-update-domain-count 2
+
+        az network nic create \
+            --name $AZURE_NIC_NAME-$UBUNTU_RELEASE \
+            --resource-group $AZURE_RESOURCE_GROUP \
+            --network-security-group $AZURE_NETWORK_SECURITY_GROUP \
+            --vnet-name $AZURE_VNET_NAME \
+             --accelerated-networking true \
+            --subnet sruSubnet \
+            --private-ip-address-version IPv4 \
+            --lb-address-pools dsLbBackEndPool_v4_$UBUNTU_RELEASE  \
+            --lb-name dsLB_$UBUNTU_RELEASE  \
+            --public-ip-address vmPublicIP-IPv4-$UBUNTU_RELEASE
+
+        az network nic create \
+            --name ${AZURE_NIC_NAME}2-$UBUNTU_RELEASE \
+            --resource-group $AZURE_RESOURCE_GROUP \
+            --network-security-group $AZURE_NETWORK_SECURITY_GROUP \
+            --vnet-name $AZURE_VNET_NAME \
+            --accelerated-networking true \
+            --subnet sruSubnet \
+            --private-ip-address-version IPv4
+
+        # Add IPv6 on primary nic
+        az network nic ip-config create \
+            --name $AZURE_NIC_NAME-$UBUNTU_RELEASE-v6 \
+            --nic-name $AZURE_NIC_NAME-$UBUNTU_RELEASE \
+            --resource-group $AZURE_RESOURCE_GROUP \
+            --vnet-name $AZURE_VNET_NAME \
+            --subnet sruSubnet \
+            --lb-address-pools dsLbBackEndPool_v6_$UBUNTU_RELEASE  \
+            --lb-name dsLB_$UBUNTU_RELEASE \
+            --private-ip-address-version IPv6
+
+    fi
+    az vm show --name $name -g $AZURE_RESOURCE_GROUP > /dev/null 2>&1 ||
+        az vm create --name=$name \
+            --image=$image_version:latest \
+            --admin-username=root --admin-username=ubuntu \
+            -g $AZURE_RESOURCE_GROUP --ssh-key-value $SSH_KEY \
+            $AZURE_BOOT_DIAG_ARG \
+            --custom-data $SAMPLE_CLOUDCONFIG $nics
+    IP=$(az vm list-ip-addresses --name $name | jq -r '.[] | .virtualMachine.network.publicIpAddresses[].ipAddress'| awk '{printf "ubuntu@%s", $1}')
+
+    echo "Created $name: $IP"
+
+    sleep 30
+    retry_ssh_command $IP "cloud-init status --wait --long" 5
+    ssh "${SSHOPTS[@]}" $IP -- cloud-init status --wait --long
+    echo "--- TEST: EXPECT NO KVP dump logs in cloud-init.log" #bbsw
```

Comment:
Modifications to typical azure-sru script to validate cloud-int https://github.com/canonical/cloud-init/pull/529